tailwind.config.js

```diff
@@ -6,18 +6,9 @@ module.exports = {
     fontFamily: theme.fonts.body,
     fontSize: theme.fontSize,
     colors: theme.colors,
-    borderRadius: theme.borderRadius,
+    borderRadius: theme.radii,
```

Comment:
not sure how this is a better name 🤔

Reply:
> Use `radii` key name to align with [System UI theme specification](https://theme-ui.com/theme-spec/)

Just want to follow the System UI theme spec. 👍

Reply:
okay!